wscl-issues/draft/simple-array-clearer-wording

```diff
@@ -0,0 +1,68 @@
+Issue:          SIMPLE-ARRAY-CLEARER-WORDING
+Forum:          Cleanup
+Category:       CLARIFICATION
+Status:         draft
+Edit History:   18-May-24, Version 1 by Jin-Cheng Guu
+References:     SIMPLE-ARRAY
+
+Problem Description:
+
+  In the draft ANSI Common Lisp specification, the note section regarding the
+  type SIMPLE-ARRAY [a] could benefit from clearer wording.
+
+    [a]: It is implementation-dependent whether displaced arrays, vectors with
+    fill pointers, or arrays that are actually adjustable are simple arrays.
+
+  Although this is logically correct, readers might mistakenly perceive a
+  contradiction with the description of SIMPLE-ARRAY [b].
+
+    [b]: The type of an array that is not displaced to another array, has no
+    fill pointer, and is not expressly adjustable is a subtype of type
+    simple-array.
+
+Proposal:
```

Comment:
The Proposal section needs a name wrapped in parenthesis.

Reply:
Add a name `(SIMPLE-ARRAY-CLEARER-WORDING:RECALL-SUBTYPE-DEFINITION)`.